infinitered/react-native-mlkit
# updates app-icon, splash-logo, github banner

## Changes

As per @mazenchami request via DMs, this PR:

- Adds app-icon via expo config.
- Adds splash-screen logo via expo config.
- Updates github banner to be more spicy.

## Notes

The logo used is the one generated by @GantMan . I vectorized it and got rid of the gradients.
Figma designs can be [found here](https://www.figma.com/design/wxBmhqPR7L6HwYU9MxfKsb/React-Native-MLKit?t=JY9lTBivqS5Fu4vv-1).

## How to Verify:

1. Uninstall current app from device.
2. `cd ./apps/ExampleApp`
3. `yarn prebuild:clean`
4. `npx expo run:android -d --no-build-cache`
5. `npx expo run:ios -d --no-build-cache`
6. Observe app-icon. Observe splash screen. Close app. Approve PR.

## Proofs

| Android | iOS  |
|---------|------|
| <video src="https://github.com/infinitered/react-native-mlkit/assets/1775841/d66b29c7-7e79-4744-ab09-02fa2aa95905" />    |  <video src="https://github.com/infinitered/react-native-mlkit/assets/1775841/67cbbc34-c0fe-46c3-97e9-e8d3b3b6ac4c" /> |

GH Banner

![yulolimum-capture-2024-06-10--19-24-54](https://github.com/infinitered/react-native-mlkit/assets/1775841/a57fc144-b50f-47f1-b585-9cc37af94fba)
